src/main/java/gyro/google/storage/BucketResource.java

```diff
@@ -552,7 +605,20 @@ public void doDelete(GyroUI ui, State state) throws Exception {
     }
 
     @Override
-    public void copyFrom(Bucket model) {
+    public void copyFrom(Bucket model) throws Exception {
+        Storage storage = createClient(Storage.class);
+
+        setIamPolicy(null);
+        Policy iamPolicy = storage.buckets()
+            .getIamPolicy(getName())
+            .set("optionsRequestedPolicyVersion", 3)
+            .execute();
+        if (iamPolicy != null) {
+            BucketIamPolicy bucketIamPolicy = newSubresource(BucketIamPolicy.class);
+            bucketIamPolicy.copyFrom(iamPolicy);
+            setIamPolicy(bucketIamPolicy);
+        }
+
```

Comment:
Move this to the end of the copyFrom()
1. As it uses the `name` field to fetch the iam policy, So if called from an external query, the `name` field would not be set and will be null and cause an error.
2. when adding things to an existing method it is a good practice to add things at the end unless there is need to add it before existing code.